#!/usr/bin/python3

#  raises a name exception with a message
def raise_exception_msg(message=""):
    '''raises a name exception with a message'''
    raise NameError(message)
